functions/Remove-DbaCredential.ps1

```diff
@@ -0,0 +1,127 @@
+function Remove-DbaCredential {
+    <#
+    .SYNOPSIS
+        Removes SQL credential(s).
+
+    .DESCRIPTION
+        Removes the SQL credential(s) that have passed through the pipeline.
+        If not used with a pipeline, Get-DbaCredential will be executed with the parameters provided
+        and the returned SQL credential(s) will be removed.
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances.
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternative credentials. Accepts PowerShell credentials (Get-Credential).
+        Windows Authentication, SQL Server Authentication, Active Directory - Password, and Active Directory - Integrated are all supported.
+        For MFA support, please use Connect-DbaInstance.
+
+    .PARAMETER Credential
+        Specifies one or more SQL credential(s) to delete. If unspecified, all accounts will be removed.
+
+    .PARAMETER ExcludeCredential
+        Specifies one or more SQL credential(s) to exclude.
+
+    .PARAMETER Identity
+        Only include specific identities
+        Note: if spaces exist in the credential identity, you will have to type "" or '' around it.
```

Comment:
To me this is assumed in PowerShell scripting, we do not include this note on other commands that have the same requirement. I'd remove this and add an example showing it.

Reply:
I added those randomly on commands that are likely to have spaces. It used to be more important back in the day when PS3 didn't support spaces in auto-complete. I'd say it's better to have it than not.

the parametersets look great! i'll merge and release shortly 👍🏼